src/components/partner/partner.service.ts

```diff
@@ -141,6 +142,15 @@ export class PartnerService {
       );
     }
 
+    if (languagesOfConsulting) {
+      await this.repo.updateRelationList({
+        id: object.id,
+        relation: 'languagesOfConsulting',
+        label: 'Language',
```

Comment:
I'll defer to @CarsonF, but I'm not sure this `label` is needed?

Reply:
No and it's actually the other label. So it'll be `Partner` here by default, which is what we want. I didn't allow giving the label of the other side. I just assumed `BaseNode` would be fine. Technically the caller could give a valid id for a different resource, that would be accepted and saved here. So that's probably not the best. But that would have to be from API directly, UI won't allow. I wasn't too worried about it because EdgeDB will validate this better anyways.